rules/sqlperf-avoid-implicit-type-conversions/rule.md

```diff
@@ -0,0 +1,38 @@
+---
+seoDescription: Do you avoid implicit data type conversions in SQL queries?
+type: rule
+archivedreason:
+title: Do you avoid implicit data type conversions in SQL queries?
+guid: 20eee575-a2ea-4283-9462-681f6ffc85d5
+uri: sqlperf-avoid-implicit-type-conversions
+created: 2024-07-23T07:26:54.0000000Z
+authors:
+  - title: Bryden Oliver
+    url: https://ssw.com.au/people/bryden-oliver
+related:
+  - sqlperf-reduce-table-size
+  - sqlperf-select-required-columns
+  - sqlperf-verify-indexes-used
+  - sqlperf-avoid-implicit-type-conversions
+  - sqlperf-use-and-instead-of-or
+  - sqlperf-minimise-large-writes
+---
+
+Specifying the wrong data types in SQL queries can make the server scan your whole table. That can take ages. Find out more.
```

Comment:
@brydeno 

I don't think we need the sentence "Find out more" here... it's not a SSW Rule thing...

Reply:
Removed.